ovs/lib/helpers/toolbox.py

```diff
@@ -197,3 +197,24 @@ def change_service_state(client, name, state, logger=None):
         else:
             logger.debug('  {0:<15} - Service {1} {2}'.format(client.ip, name, action))
             print '  [{0}] {1} {2}'.format(client.ip, name, action)
+
+    @staticmethod
+    def wait_for_service(client, name, status=True):
+        """
+        Wait for service to enter status
+        :param client: SSHClient to run commands
+        :param name: name of service
+        :param status: True - running/False - not running
+        """
+        ServiceManager.enable_service(name, client=client)
+        tries = 10
+        while tries > 0:
+            service_status = ServiceManager.get_service_status(name, client)
+            if service_status == status:
+                break
+            print('... waiting for service')
```

Comment:
I assume it's too late to get rid of this print and replace it by logger.whatever like we do at every other place?